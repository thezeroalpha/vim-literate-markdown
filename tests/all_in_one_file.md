# Different languages, all tangled to the same file
<!-- :Tangle DIRNAME/FILENAME.out -->
This file is a collection of different languages.
They should all end up in the same output file.

Some shell:

```sh
echo "Hello!"
```

Some Ruby:

```ruby
def a(x)
  x*2
end
```

And more ruby:

```ruby
a(3)
```
