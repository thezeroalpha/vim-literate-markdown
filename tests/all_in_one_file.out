echo "Hello!"

def a(x)
  x*2
end

a(3)

